### DISCUSSION TITLE
Chat channel e.g. discord

### DISCUSSION THREAD

**Discussion by rufuspollock** [flowershow/flowershow#186] [General and Help] [answered]
Currently we have  these github discussions but wondering if a public chat would be valuable.

**Comment by olayway** [comment]
@rufuspollock I think it's a great idea. We would be able to address any issues and questions people might have much faster. Plus, it would shorten the distance between the users and us - the developers' team - which is excellent.

**Reply by olayway** [reply]
I can set it up if we decide to go for it.

**Comment by rufuspollock** [comment]
@olayway any thoughts on chat system. Three options in my mind are:

- Discord:
  -  pros: very well known especially in tech community, many people have accounts, easy to sign up etc. Obsidian use it
  - cons: proprietary etc (what is long-term viability)
- Element / Matrix (element.io)
  - Pros: open-source (mostly), very slick, truly decentralized/distributed etc etc
  - Cons: less people know it, IIRC to sign up you need to have your public/private key. as slightly more complex UX is more complex
- GChat: not sure this even possible as not sure there is a way to provide an invite link (think people have to be explicitly invited)

My vote would be for discord for now - you can maybe bridge to matrix later anyway https://matrix.org/docs/projects/bridge/matrix-discord-bridge

Interested in what you think - and others!

**Reply by olayway** [reply]
I would also choose Discord for now. It's the most popular solution and whereas Element may be decentralized/non-proprietary I don't think this is of major concern for our use case. I think GChat is the worst option - the UX is terrible IMO; you can't even reply to a message in a thread...among other things you can't do.

**Comment by rufuspollock** [comment]
We are going with discord and here's the tracking issue https://github.com/flowershow/flowershow/issues/203

**UPDATE: here's the channel https://discord.gg/vQ5Y2uUzt6**

**Comment by PhilippeduPreez** [comment, marked as answer]
We are going with discord and here's the tracking issue https://github.com/flowershow/flowershow/issues/203
Updated channel link https://discord.gg/vQ5Y2uUzt6
